store/database.go

```diff
@@ -732,7 +732,7 @@ func (*Store) listDatabaseImplV2(ctx context.Context, tx *Tx, find *FindDatabase
 	if v := find.InstanceID; v != nil {
 		where, args = append(where, fmt.Sprintf("instance.resource_id = $%d", len(args)+1)), append(args, *v)
 	}
-	if v := find.DatabaseName; v != nil {
```

Comment:
Prefer to use `DatabaseName` instead of `DatabaseID` as we’re using the name in the query

Reply:
Done.